### PR TITLE
Fixed height issue of dynamic config dialog where the user cannot scroll

### PR DESCRIPTION
**Pull request by shuqi7**
This is to fix a bug in coordinator dynamic config dialog which is too long and user cannot scroll down.

Bug:
image

Fixed:
image

### CODE REVIEW

File: web-console/src/dialogs/coordinator-dynamic-config.scss [hunk 1]
```diff
@@ -17,15 +17,17 @@
  */
 
 .coordinator-dynamic-config {
-  .bp3-dialog-body {
+  margin-top: 5vh;
+
+  .pt-dialog-body {
```
Comment (vogievetsky) [inline web-console/src/dialogs/coordinator-dynamic-config.scss]: good one, must have missed that in the blueprint 3 -> blueprint 1 migration

**Comment by shuqi7** [comment]
> Searching codebase for 'bp3', I also see a
> 
> ```
> .retention-dialog {
>   width: 750px;
> 
>   .dialog-body {
>     overflow: scroll;
>     max-height: 70vh;
> 
>     .bp3-form-group {
>       margin: 0 0 5px;
>     }
> ...
> ```
> in `web-console/src/dialogs/retention-dialog.scss`
> 
> Unsure if it matters for anything 🙃

Will test and fix that

**Comment by shuqi7** [comment]
> > Searching codebase for 'bp3', I also see a
> > ```
> > .retention-dialog {
> >   width: 750px;
> > 
> >   .dialog-body {
> >     overflow: scroll;
> >     max-height: 70vh;
> > 
> >     .bp3-form-group {
> >       margin: 0 0 5px;
> >     }
> > ...
> > ```
> > in `web-console/src/dialogs/retention-dialog.scss`
> > Unsure if it matters for anything 🙃
> 
> Will test and fix that

The last `bp3` class is removed in the PR #7276 when fixing bugs in retention cell and retention dialog

**Comment by clintropolis** [comment]
>The last bp3 class is removed in the PR #7276 when fixing bugs in retention cell and retention dialog

Thanks for looking :+1:

**Pull request merged**